DataLoaderUtil_example.ipynb

```diff
@@ -31,7 +29,7 @@
     "databin = dataloader.ClassificationDataset(infer_list, dataset_root=dataset_root, binary=True)\n",
     "print(\"Amount of images\", len(databin))\n",
     "num_classes_binary = databin.label_list.shape[1]\n",
-    "for img in tqdm(databin):\n",
+    "for img in tqdm(databin, leave=False):\n",
```

Comment:
not a relevant change for the end user (in all cases)

Reply:
Ok, modifico